lib/theme/variables.css

```diff
@@ -3,6 +3,8 @@
   --klima-blue: #0ba1ff;
   --gray: rgb(160, 160, 160);
   --white: #ffffff;
+  --surface-04: #767676;
+  --surface-05: #a1a1a1;
```

Comment:
these need to have dark mode variants and be a part of .light and .dark class.

Since they are only used for text lets call them
```
--font-04
--font-05
```

If they are very very similar to --font-01, 02 or 03 then just use those colors instead and remove them altogether.